src/lib.rs

```diff
@@ -244,7 +278,7 @@ impl<'object, Endian> DebugInfo<'object, Endian>
                                    &debug_line,
                                    &debug_str,
                                    &header,
-                                   with_functions);
+                                   opts.with_functions);
```

Comment:
To be consistent, this should probably be `opts`, even if we don't need the `with_demangling` information.